Lib/profile/sample.py

```diff
@@ -47,8 +49,14 @@ def sample(self, collector, duration_sec=10):
                 try:
                     stack_frames = self.unwinder.get_stack_trace()
                     collector.collect(stack_frames)
-                except (RuntimeError, UnicodeDecodeError, OSError):
```

Comment:
This is wrong: the profiler can and will fail naturally with OSError

Reply:
I had to modify the windows part of remote debugging in dbe2c0a to not raise a generic `OSError` so we can catch it properly